crates/ruff_linter/src/rules/pylint/rules/too_many_arguments.rs

```diff
@@ -53,7 +53,7 @@ impl Violation for TooManyArguments {
     #[derive_message_formats]
     fn message(&self) -> String {
         let TooManyArguments { c_args, max_args } = self;
-        format!("Too many arguments to function call ({c_args} > {max_args})")
+        format!("Too many arguments to function definition ({c_args} > {max_args})")
```

Comment:
Should probably say "in" instead of "to" now
```suggestion
        format!("Too many arguments in function definition ({c_args} > {max_args})")
```